logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnAnsweredCall.kt

```diff
@@ -4,20 +4,24 @@ import com.sun.jna.Pointer
 import com.wire.kalium.calling.callbacks.AnsweredCallHandler
 import com.wire.kalium.logic.callingLogger
 import com.wire.kalium.logic.data.call.CallRepository
+import com.wire.kalium.logic.data.id.QualifiedIdMapper
 import com.wire.kalium.logic.feature.call.CallStatus
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
-//TODO(testing): create unit test
+// TODO(testing): create unit test
```

Comment:
Should we create now these unit tests then?

Reply:
Yeah later in a separate PR would be better, I am just refactoring qualifiedId parsing here.